src/commands/emote.js

```diff
@@ -28,7 +31,43 @@ module.exports = {
 					logger.info(`Emote attached: ${response}`);
 				}
 			} catch (error) {
-				logger.error(`Emote failed to send: ${error}`);
+				sendErrorMessage(`Emote failed to send: ${error}`, message.channel);
+			}
+
+			try {
+				await message.delete();
+			} catch (error) {
+				logger.error(`Error deleting the message: ${error}`);
+			}
+		}
+	},
+	mEmote : {
+		name : 'm-emote',
+		args : true,
+		description : 'Sends a bigger version of an emote',
+		execute: async(message, data) => {
+			logger.info('multi-emote command...');
+			const emojis = data.filter(possibleEmoji => {
+				return (possibleEmoji[0] === '<' && possibleEmoji.slice(-1) === '>');
+			}).map(emoji => {
+				let fileType;
+				if (emoji[1] === 'a') {
+					fileType = '.gif';
+					emoji = emoji.replace('a', '');
+				} else {
+					fileType = '.png';
+				}
+				const emojiId = parseEmojiText(emoji);
+				return `https://cdn.discordapp.com/emojis/${emojiId + fileType}`;
```

Comment:
If we get more URLs in the future, we might want to start making a constants file where we import, incase there becomes multiple instances of this, but this is just a nice to have and i dont reall give a fuk

Reply:
Good point